src/flb_config.c

```diff
@@ -781,6 +781,9 @@ static int configure_plugins_type(struct flb_config *config, struct flb_cf *cf,
                 continue;
             }
 
+            // mark as error by default.
```

Comment:
Please change the comment type and add a brief of what it's objective is, otherwise it's useless. 

The way I saw it it's something like "Set ret to -1 to ensure that we treat any unhandled plugin or value types as errors".

Otherwise just remove the comment.